region/maridia/inner-pink/Botwoon Energy Tank Room.json

```diff
@@ -2415,8 +2482,18 @@
       "link": [6, 6],
       "name": "Zoa Farm",
       "requires": [
-        {"refill": ["Energy"]}
-      ]
+        {"or": [
+          {"and": [
+            "Grapple",
+            {"cycleFrames": 285}
+          ]},
+          {"and": [
+            "canDodgeWhileShooting",
```

Comment:
I can't reliably farm all 5 without getting hit

Reply:
I would like to see `Quick Underwater Turnaround` tech: canMidairWiggle
Or a beam so you can kill the bugs close to samus: wave or spazer.  plasma would reduce number of farms you can reach